### ISSUE TITLE
Design feedback for Staging Review: Estimate your benefits updates in the Comparison Tool

### ISSUE THREAD

**Issue by CrystabelReiter** [department-of-veterans-affairs/va.gov-team#10179]
**Do:** Mandatory feedback
**Try:** Exploratory next step
**Consider:** Sharing ideas/alternate ways of thinking

Looking forward to hearing about your mobile testing suggestions/takeaways! 

**Do:**
- No "do" feedback

**Try:**

- No "try" feedback

**Consider:**

- For when a user selects, “Post GI 9/11 Bill Ch 33” from the question “Which GI Bill benefit do you want to use?”, consider adding a warning icon to the yellow background warning. Similar to the blue background alert that appears when a user clicks V&RE. It’s helpful to not rely on color alone to convey context. 

**Definition of Done:**
- [x] Incorporate the **Do** feedback, and let us know on this ticket when they are updated mockups to check out 

- [ ] Leave any comments for feedback you decide _not_ to take

Emily or Crystabel will close the ticket once those things are all set. Thank you!

**Comment by amyknox** [comment]
@CrystabelReiter - closing the loop on older stories.  Do you have a screenshot of where this is happening? I'm not able to reproduce that specific issue.

**Comment by CrystabelReiter** [comment]
@amyknox Here is the screenshot that I found in my old notes: 
<img width="517" alt="Screen Shot 2020-08-12 at 10 50 59 AM" src="https://user-images.githubusercontent.com/18386248/90030189-c01d5700-dc89-11ea-9541-946dca932288.png">

**Comment by ccruzgra** [comment]
@CrystabelReiter @amyknox Looks like the yellow warning alert is similar to these "Background color only" alerts: https://design.va.gov/components/alertboxes#background-color-only

In that case, maybe the blue one is the one that shouldn't have the icon? Although, I agree icons are helpful in conveying meaning. Maybe the background color only alerts need to be revised.

**Comment by CrystabelReiter** [comment]
@ccruzgra True, I think right now adding the icon is optional with these background alerts. I think we can keep the alerts as they are here. 

I can bring up this alert question to the design system council and see if we'd like to change the documentation on these bg alerts to be more clear whether we should use icons or not.

**Comment by amyknox** [comment]
@CrystabelReiter - what do you think the turnaround on the query to design council might be? Ideally we can get the answer quickly and make an update so the alerts are consistent.

**Comment by CrystabelReiter** [comment]
@amyknox Can I get you an answer by end of day?

**Comment by ccruzgra** [comment]
@CrystabelReiter Sometimes I've noticed confusion between which is which for the featured content component vs. the blue background color only alert since they are a similar color:

https://design.va.gov/components/featured-content

**Comment by CrystabelReiter** [comment]
Talked to Ryan T (visual design lead at DEPO) and got an answer!  He thinks we should add the :warning: icon to the GI Bill alert. The yellow alert seems severe enough to call attention to it differently than the VR&E message.

@ccruzgra I've seen that confusion as well. I made a ticket around alerts for the design system consistency audit that addresses that.
